pytest/tests/sanity/state_sync2.py

```diff
@@ -4,16 +4,22 @@
 # and sync
 
 import sys, time
+import fcntl
 
 sys.path.append('lib')
 
 from cluster import start_cluster
 from utils import LogTracker
 
+fcntl.fcntl(1, fcntl.F_SETFL, 0) # no cache when execute from nightly runner
```

Comment:
What is this line doing? Disabling buffering?

Reply:
I don't remember exactly, it worked to disable buffering in subprocess.PIPE kind of stdout/stderr in gitlab-ci parallel-run-test.py (which is also when a pytest launched by nightly.py)